projects/requisition-management/src/app/models/requisition/requisition.model.ts

```diff
@@ -7,7 +7,7 @@ import { User } from 'ish-core/models/user/user.model';
 
 export type RequisitionStatus = 'PENDING' | 'APPROVED' | 'REJECTED';
 
-export type RequisitionViewer = 'buyer' | 'approver';
+export type RequisitionViewer = 'buyer' | 'approver' | 'both';
```

Comment:
'all' or an optional view (no view for all) would be better, wouldn't be?

Reply:
since the 'both' type comes with some very specific behaviour, I would rather have it be a seperate option. I will change it to 'all'